Client/pom.xml

```diff
@@ -156,6 +177,55 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <!-- create a windows .exe with everything in it -->
+            <plugin>
+                <groupId>com.akathist.maven.plugins.launch4j</groupId>
+                <artifactId>launch4j-maven-plugin</artifactId>
+                <version>1.7.25</version>
+                <executions>
+                    <execution>
+                        <id>l4j-cli</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>launch4j</goal>
+                        </goals>
+                        <configuration>
+                            <headerType>gui</headerType>
+                            <outfile>target/CardBoard.exe</outfile>
+                            <jar>target/${project.build.finalName}-shaded.jar</jar>
+                            <classPath>
+                                <mainClass>org.projectcardboard.client.Launcher</mainClass>
+                            </classPath>
+                            <icon>src/main/resources/icon.ico</icon>
+                            <downloadUrl>https://adoptopenjdk.net/?variant=openjdk11</downloadUrl>
+                            <jre>
+                                <minVersion>11</minVersion>
+                                <jdkPreference>preferJre</jdkPreference>
+                            </jre>
+                            <versionInfo>
+                                <fileVersion>1.0.0.0</fileVersion>
+                                <txtFileVersion>${project.version}</txtFileVersion>
+                                <fileDescription>${project.name}</fileDescription>
+                                <copyright>2020 Project CardBoard</copyright>
+                                <productVersion>1.0.0.0</productVersion>
+                                <txtProductVersion>1.0.0.0</txtProductVersion>
+                                <productName>${project.name}</productName>
+                                <companyName>Project CardBoard</companyName>
+                                <internalName>Project CardBoard</internalName>
+                                <originalFilename>CardBoard.exe</originalFilename>
+                            </versionInfo>
+                            <messages>
+                                <startupErr>An error occurred while starting the application.</startupErr>
+                                <bundledJreErr>This application was configured to use a bundled Java Runtime Environment but the runtime is missing or corrupted.</bundledJreErr>
```

Comment:
I thought we were going with the download link approach.  In which case, shouldn't the link be herre?

Reply:
Definitely, forgot to update that.